### PR TITLE
Admissions Search Page

### PR DESCRIPTION
**Pull request by brian-yang**
Summary: I implemented the search filter with hackers retrieved from the api on the admissions page. The hackers can be filtered by their github usernames and the wave number. 

Test: I went to localhost:3000/admissions and typed stuff into the search bar. #31
admissions
)

### CODE REVIEW

**Comment by brian-yang** [comment]
@jc3m Ready for review

File: src/services/api/registration.js [hunk 1]
```diff
@@ -14,3 +14,17 @@ export function fetchRegistration(id, token) {
...
+    headers: {
+      Accept: 'application/json',
+      Authorization: token,
+    },
+  }).then(response => {
+    if (response.status >= 400) {
+      throw new Error(response);
+    }
+    return response.json()
+  });
```
Comment (jc3m) [inline src/services/api/registration.js]: I'm getting a build error here. I think you're missing a closing curly brace?

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # See https://help.github.com/ignore-files/ for more about ignoring files.
 
+# package-lock
+package-lock.json
```
Comment (jc3m) [inline .gitignore]: We're using yarn for this project so we shouldn't be seeing `package-lock.json` anyway. If it's in your directory, just delete it.

File: src/App.js [hunk 3]
```diff
@@ -23,6 +24,7 @@ class App extends Component {
             <Switch>
               <Route exact path="/" component={Home} />
               <Route path="/login" component={Login} />
+              <Route path="/admissions" component={Admissions} />
               <Route path="/events" component={Events} />
               <Route path="/profile/:handle" component={Profile} />
             </Switch>
```
Comment (jc3m) [inline src/App.js]: Can you add an entry in the side menu bar for the admissions page?

**Comment by brian-yang** [comment]
@jc3m Ok made all the requested changes

**Pull request merged**